src/crimson/net/SocketConnection.cc

```diff
@@ -44,9 +44,8 @@ namespace {
 }
 
 SocketConnection::SocketConnection(SocketMessenger& messenger,
-                                   const entity_addr_t& my_addr,
                                    Dispatcher& dispatcher)
-  : Connection(my_addr),
+  : Connection(),
```

Comment:
just out of curiosity, why shall we call the parent constructor explicitly?

Reply:
My bad habit from Python... will remove it.

Reply:
done.